sitepipe.cabal

```diff
@@ -23,30 +23,28 @@ library
                      , SitePipe.Parse
                      , SitePipe.Types
                      , SitePipe.Utilities
-  build-depends:       base                 >= 4.9.0.0  && < 4.11.0.0
-                     , optparse-applicative >= 0.13.2
-                     , unordered-containers >= 0.2.8.0  && < 0.3
-                     , directory            >= 1.3.0.0  && < 1.4
-                     , filepath             >= 1.4.1.1  && < 1.5
-                     , megaparsec           >= 5.3.1    && < 6.6
-                     , mtl                  >= 2.2.1    && < 2.3
-                     , pandoc               >= 1.19.2   && < 1.20
-                     , yaml                 >= 0.8.23.3 && < 0.9
-                     , mustache             >= 2.2.3
-                     , bytestring           >= 0.10.8.1 && < 0.11
-                     , text                 >= 1.2.2.2  && < 1.3
-                     , parsec               >= 3.1.11   && < 3.2
-                     , exceptions           >= 0.8.3    && < 0.9
-                     , Glob                 >= 0.8.0    && < 0.10
-                     , lens-aeson           >= 1.0.2    && < 1.1
-                     , lens                 >= 4.15.4   && < 4.17
-                     , aeson                >= 1.1.2.0  && < 1.3
-                     , shelly               >= 1.6.8    && < 1.8
-                     , MissingH             >= 1.4.0.1  && < 1.5
-                     , containers           >= 0.5.7.1  && < 0.6
-
-  if impl(ghc >= 8.2)
-    build-depends:     pandoc >= 1.19.2.4
+  build-depends:       base
```

Comment:
Non-stack folks tend to get a bit upset about not having any version bounds at all; even if we just had `>= <the-lts-version>` it would probably be good enough.

Reply:
is there a tool to do this? i am not going to do it by hand.

Reply:
There's `cabal gen-bounds` but it ignores the LTS; I just discovered the `--pvp-bounds` flag for stack upload which gives us this for free when shipping to hackage so I'll just use that 👍